Rocket.Chat/Managers/ShortcutsManager.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ShortcutsManager.swift
+//  Rocket.Chat
+//
+//  Created by Artur Rymarz on 12.07.2018.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+struct ShortcutServerItem {
+    let name: String
+    let url: String
+    let index: Int
+}
+
+struct ShortcutsManager {
+    static let addServerActionIdentifier = "addserver"
+    static let connectServerNavIdentifier = "ConnectServerNav"
+    static let serverIndex = "kServerIndex"
+
+    private static var shortcuts: [ShortcutServerItem]? {
+        let servers = DatabaseManager.servers?.enumerated().compactMap({ index, server in
+            ShortcutServerItem(name: server[ServerPersistKeys.serverName] ?? "", url: server[ServerPersistKeys.serverURL] ?? "", index: index)
+        })
+
+        return servers
+    }
+
+    static func sync() {
+        UIApplication.shared.shortcutItems = shortcuts?.compactMap({ shortcut in
+            let url = URL(string: shortcut.url)?.host ?? shortcut.url
+            return UIMutableApplicationShortcutItem(type: url, localizedTitle: shortcut.name, localizedSubtitle: url, icon: nil, userInfo: [serverIndex: shortcut.index])
+        })
+    }
+
+    static func selectServer(at index: Int) {
```

Comment:
`ServersListView` has pretty much the same method, should we extract it somewhere? Does it fit in `DatabaseManager` or `ServerManager`?

Reply:
Yeah... we could've use this one too maybe? `AppManager.changeSelectedServer()`. I'm going to merge it and then we can refactor in another PR, to avoid the conflicts for you.